src/main/scala/github4s/Decoders.scala

```diff
@@ -19,4 +19,150 @@ object Decoders {
+      releases_url ← c.downField("releases_url").as[Option[String]]
+      deployments_url ← c.downField("deployments_url").as[Option[String]]
+    } yield Repository(
+      id           = id,
+      name         = name,
+      full_name    = full_name,
+      owner        = owner,
+      `private`    = priv,
+      description  = description,
+      fork         = fork,
+      created_at   = created_at,
+      updated_at   = updated_at,
+      pushed_at    = pushed_at,
+      homepage     = homepage,
+      language     = language,
+      organization = organization,
+      status       = RepoStatus(
+        size              = size,
+        stargazers_count  = stargazers_count,
+        watchers_count    = watchers_count,
+        forks_count       = forks_count,
+        open_issues_count = open_issues_count,
+        open_issues       = open_issues,
+        watchers          = watchers,
+        network_count     = network_count,
+        subscribers_count = subscribers_count,
+        has_issues        = has_issues,
+        has_downloads     = has_downloads,
+        has_wiki          = has_wiki,
+        has_pages         = has_pages
+      ),
+      urls         = RepoUrls(
+        url               = url,
+        html_url          = html_url,
+        git_url           = git_url,
+        ssh_url           = ssh_url,
+        clone_url         = clone_url,
+        svn_url           = svn_url,
+        forks_url         = forks_url,
+        keys_url          = keys_url,
+        collaborators_url = collaborators_url,
+        teams_url         = teams_url,
+        hooks_url         = hooks_url,
+        issue_events_url  = issue_events_url,
+        events_url        = events_url,
+        assignees_url     = assignees_url,
+        branches_url      = branches_url,
+        tags_url          = tags_url,
+        blobs_url         = blobs_url,
+        git_tags_url      = git_tags_url,
+        git_refs_url      = git_refs_url,
+        trees_url         = trees_url,
+        statuses_url      = statuses_url,
+        languages_url     = languages_url,
+        stargazers_url    = stargazers_url,
+        contributors_url  = contributors_url,
+        subscribers_url   = subscribers_url,
+        subscription_url  = subscription_url,
+        commits_url       = commits_url,
+        git_commits_url   = git_commits_url,
+        comments_url      = comments_url,
+        issue_comment_url = issue_comment_url,
+        contents_url      = contents_url,
+        compare_url       = compare_url,
+        merges_url        = merges_url,
+        archive_url       = archive_url,
+        downloads_url     = downloads_url,
+        issues_url        = issues_url,
+        pulls_url         = pulls_url,
+        milestones_url    = milestones_url,
+        notifications_url = notifications_url,
+        labels_url        = labels_url,
+        releases_url      = releases_url,
+        deployments_url   = deployments_url
```

Comment:
Because we are in the `urls` class, it makes sense to remove the `_url` suffix from all fields?